src/dagre/position/bk.js

```diff
@@ -129,18 +131,34 @@ function findOtherInnerSegmentNode(g, v) {
   }
 }
 
+/**
+ * Sets `conflicts[v][w] = true`, creating objects if needed.
+ *
+ * @param {{[nodeId: string | number]: {[nodeId: string | number]: true}}} conflicts - Object to set.
+ * @param {string | number} v - First Node ID
+ * @param {string | number} w - Second Node ID
+ */
 function addConflict(conflicts, v, w) {
   if (v > w) {
     var tmp = v;
     v = w;
     w = tmp;
   }
 
-  var conflictsV = conflicts[v];
-  if (!conflictsV) {
-    conflicts[v] = conflictsV = {};
+  if (!Object.prototype.hasOwnProperty.call(conflicts, v)) {
+    // can't use conflicts[v] = {} since it's unsafe if v = `__proto__`
+    Object.defineProperty(conflicts, v, {
+      enumerable: true,
+      configurable: true,
+      value: {},
+    });
   }
-  conflictsV[w] = true;
+  var conflictsV = conflicts[v];
+  Object.defineProperty(conflictsV, w, {
+    enumerable: true,
+    configurable: true,
+    value: true,
```

Comment:
```suggestion
    value: true,
    writable: true,
```

It might be worth adding `writable: true` here in case any dependencies are using `addConflict()` directly.

Without it, dagre will be fine. The only time we assign to these values is in this `_.merge` command, but since these values are only even `true`, that's not an issue: https://github.com/tbo47/dagre-es/blob/b4cd721ba364903459a4d5cf1c65e9cbb7ee7117/src/dagre/position/bk.js#L345